db/migrations/04_alter_spacess_to_add_price_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE spaces ADD COLUMN price FLOAT;
```

Comment:
price should be an `integer` not a `float`. Then do all calculations in pence. The reason for this is that with floats you can have fractional pennies which wouldn't typically be something you'd want. 

Floats are harder to do calculations with than integers. If you do calculations in pence it is easy to get the correct value by dividing the calculations by 100.